libp2p/pubsub/pubsub.py

```diff
@@ -137,6 +141,7 @@ def __init__(
 
         # Call handle peer to keep waiting for updates to peer queue
         asyncio.ensure_future(self.handle_peer_queue())
+        asyncio.ensure_future(self.handle_dead_peer_queue())
```

Comment:
You aren't keeping track of this task I think implies that this might lead to unclean shutdowns?